src/main/java/com/atlauncher/data/Settings.java

```diff
@@ -109,6 +109,7 @@ public class Settings {
     public boolean disableLegacyLaunching = false;
     public boolean useSystemGlfw = false;
     public boolean useSystemOpenAl = false;
+    public boolean useDedicatedGpu = true;
```

Comment:
Only thing I'm not sure of is if this should be defaulted to true.

What I might do is for now I'll merge this in, and then play around on some linux machines just to make sure this defaulting to on doesn't cause any issues.

I would think we would want this on always to prevent people from not knowing it's there on linux to have issues, but yeah I'll just double check on some systems I have laying around